src/tests/ftest/launch.py

```diff
@@ -1414,14 +1419,15 @@ def install_debuginfos():
     # installation
     path = os.path.sep + os.path.join('usr', 'share', 'spdk', 'include')
     if os.path.islink(path):
-        cmds.append(["sudo", "rm", "-f", path])
+        cmd_list.append(["sudo", "rm", "-f", path])
 
     if USE_DEBUGINFO_INSTALL:
         yum_args = [
             "--exclude", "ompi-debuginfo", "libpmemobj", "python", "openmpi3"]
-        cmds.append(["sudo", "yum", "-y", "install"] + yum_args)
-        cmds.append(["sudo", "debuginfo-install", "--enablerepo=*-debuginfo",
-                     "-y"] + yum_args + ["daos-server", "gcc"])
+        cmd_list.append(["sudo", "dnf", "debuginfo-install", "-y"] + yum_args)
```

Comment:
This is switching the semantics to installing the `-debuginfo` packages for the `yum_args` list rather than the packages themselves.  This more than likely results in installing the packages themselves, but I am not entirely positive.  Have you seen the result of this?  Does it install the base package along with the `-debuginfo`?